### DISCUSSION TITLE
Add a `transparency-reduce` variant

### DISCUSSION THREAD

**Discussion by manvankaliber** [tailwindlabs/tailwindcss#13652] [Ideas]
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-transparency

If possible, please add this to v3.

**Comment by fordaaron** [comment]
There is currently a PR for this for [v4](https://github.com/tailwindlabs/tailwindcss/pull/11692).

You can add this to v3 via your `tailwind.config` with `addVariant`. [Tailwind Play Demo](https://play.tailwindcss.com/PZAH4zshhK?file=config)

```js
plugins: [
  require('tailwindcss/plugin')(function ({ addVariant }) {
    addVariant('transparency-reduce', '@media (prefers-reduced-transparency)')
  }),
],
```

**Comment by manvankaliber** [comment]
I would rather see this implemented in v3 by default though
